client.go

```diff
@@ -10,71 +10,64 @@ import (
 	"strings"
 )
 
-const (
-	defaultBaseURL = "https://%s.recurly.com/"
-)
-
-type (
-	// Client manages communication with the Recurly API.
-	Client struct {
-		// client is the HTTP Client used to communicate with the API.
-		client *http.Client
-
-		// subdomain is your account's sub domain used for authentication.
-		subDomain string
-
-		// apiKey is your account's API key used for authentication.
-		apiKey string
-
-		// BaseURL is the base url for api requests.
-		BaseURL string
-
-		// Services used for talking with different parts of the Recurly API
-		Accounts      *AccountsService
-		Adjustments   *AdjustmentsService
-		Billing       *BillingService
-		Coupons       *CouponsService
-		Redemptions   *RedemptionsService
-		Invoices      *InvoicesService
-		Plans         *PlansService
-		AddOns        *AddOnsService
-		Subscriptions *SubscriptionsService
-		Transactions  *TransactionsService
-	}
-
-	// Params are used to send parameters with the request.
-	Params map[string]interface{}
-)
+const defaultBaseURL = "https://%s.com/"
+
+// Client manages communication with the Recurly API.
+type Client struct {
+	// client is the HTTP Client used to communicate with the API.
+	Client *http.Client
+
+	// subdomain is your account's sub domain used for authentication.
+	subDomain string
+
+	// apiKey is your account's API key used for authentication.
+	apiKey string
+
+	// BaseURL is the base url for api requests.
+	BaseURL string
+
+	// Services used for talking with different parts of the Recurly API
+	Accounts      AccountsService
+	Adjustments   AdjustmentsService
+	Billing       BillingService
+	Coupons       CouponsService
+	Redemptions   RedemptionsService
+	Invoices      InvoicesService
+	Plans         PlansService
+	AddOns        AddOnsService
+	Subscriptions SubscriptionsService
+	Transactions  TransactionsService
+}
 
-// NewClient creates a new Recurly API Client.
-func NewClient(subDomain string, apiKey string, httpClient *http.Client) *Client {
+// NewClient sets the unexported fields on the struct and returns a Client.
+func NewClient(subDomain, apiKey string, httpClient *http.Client) *Client {
 	if httpClient == nil {
 		httpClient = http.DefaultClient
 	}
 
-	c := &Client{
-		client:    httpClient,
+	client := &Client{
+		Client:    httpClient,
```

Comment:
Unexported in [5238e2](https://github.com/blacklightcms/go-recurly/pull/10/commits/322b61900754b8bcf3719fcf55a8b1a7955238e2)